### PR TITLE
Upgrade reqwest to ^0.9.0

### PR DESCRIPTION
**Pull request by h-michael**
related #121

reqwest v0.8.x don't correspond to openssl 1.1.1.
So we should upgrade reqwest.
There are [some breaking changes](https://github.com/seanmonstar/reqwest/blob/master/CHANGELOG.md#breaking-changes).

refer to: https://docs.rs/reqwest/0.9.0/reqwest/struct.RequestBuilder.html#method.headers

### CODE REVIEW

File: graphql_client_cli/src/main.rs [hunk 1]
```diff
@@ -101,13 +100,8 @@ fn introspect_schema(
...
-    let mut headers = Headers::new();
-    headers.set(Accept(vec![qitem(mime::APPLICATION_JSON)]));
-
-    match authorization {
-        Some(token) => headers.set(reqwest::header::Authorization(token)),
-        None => (),
-    };
+fn construct_headers() -> HeaderMap {
+    let mut headers = HeaderMap::new();
+    headers.insert(CONTENT_TYPE, HeaderValue::from_static("application/json"));
```
Comment (tomhoule) [inline graphql_client_cli/src/main.rs]: in addition to Content-Type, we should also set the [Accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept) header (to `application/json` as well).

Comment (h-michael) [reply graphql_client_cli/src/main.rs]: @tomhoule Oops! Thank you for finding my oversight.

**Comment by h-michael** [comment]
@tomhoule I fixed it.

**Comment by tomhoule** [comment]
I'll merge as soon as CI passes

**Pull request merged**